recipe/build.py

```diff
@@ -377,7 +411,11 @@ def extract(self):
                     "--silent",
                 ]
                 check_call(cmd)
-            self.copy(tmpd)
+            try:
+                self.copy(tmpd)
+            except Exception:
+                call(["ls", "-lR", str(tmpd)])
+                raise
```

Comment:
What was the exception that came up here?

Reply:
There was a FileNotFoundException. I was trying to get some debugging info.  I can remove this try-except if you want

Reply:
Yeah that's probably a good idea or change it so it could work on Windows if it came up. `os.listdir` could work

Reply:
Yeah, I'll just remove

Reply:
Done!